src/js/tech/html5.js

```diff
@@ -549,9 +549,19 @@ class Html5 extends Tech {
    * @method setSrc
    */
   setSrc(src) {
+    let loadstartlistener = Html5.prototype.loadStartListener_;
+
+    this.off(this.el_, 'loadstart', loadstartlistener);
+    this.one(this.el_, 'loadstart', () => this.one(this.el_, 'loadstart', loadstartlistener));
```

Comment:
Also, this function literal wouldn't get cleared properly on subsequent calls to setSrc() which would cause the `loadstartlistener_` to get invoked on the wrong `loadstart`